tests/templates-static-data.spec.ts

```diff
@@ -7,13 +7,6 @@ describe('files', () => {
     })
   })
 })
-describe('artifacts', () => {
```

Comment:
Artifacts system is gone now. It didn't really make sense. We want a higher abstraction than derived files.